web/packages/teleport/src/services/history/history.test.ts

```diff
@@ -118,11 +118,48 @@ describe('services/history', () => {
         .spyOn(history, 'getRoutes')
         .mockReturnValue(['/web/login', '/current-location']);
       history.original().location.pathname = '/current-location';
-      history.goToLogin(true);
+      history.goToLogin({ rememberLocation: true });
 
       const expected =
         '/web/login?redirect_uri=http://localhost/current-location';
       expect(history._pageRefresh).toHaveBeenCalledWith(expected);
     });
+
```

Comment:
can you also add another test case to make sure we are appending the query parts to redirect url?